hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -3550,9 +3562,10 @@ func reconcileSameNamespaceNetworkPolicy(policy *networkingv1.NetworkPolicy) err
 	return nil
 }
 
-func reconcileKASNetworkPolicy(policy *networkingv1.NetworkPolicy, hcluster *hyperv1.HostedCluster, isOpenShiftMgmtCluster bool) error {
+func reconcileKASNetworkPolicy(policy *networkingv1.NetworkPolicy, hcluster *hyperv1.HostedCluster, isOpenShiftMgmtCluster bool, isEgressBroken bool) error {
```

Comment:
maybe s/isEgressBroken/specifyEgress/ ?

Reply:
I'm thinking that when the bug is backported, then we will remove the check that calls this and having this param name is going to be confusing

Reply:
I was in a bad mood.  Will fix :grin: